### DISCUSSION TITLE
报这个错请问怎么解决

### DISCUSSION THREAD

**Discussion by wangwei2919** [deepjavalibrary/djl#1896] [Q&A]
![QQ截图20220812093042](https://user-images.githubusercontent.com/66586230/184268728-d78d906f-e19b-4c69-b575-5144435578fa.png)

Expected tensor for argument #1 'indices' to have one of the following scalar types: Long, Int; but got CPUFloatType instead (while checking arguments for embedding)

**Comment by lanking520** [comment]
You are not using the right type for your indices NDArray. You can apply a `nd.toType(DataType.INT64, false)`.

**Reply by wangwei2919** [reply]
However, I run it directly with GitHub code. If I modify nd.totype (datatype.int64, false), I will also report an error, so what should I do...
![1](https://user-images.githubusercontent.com/66586230/184272586-1c6b179d-e8e1-40e3-ae4a-520f9d6f2ca2.png)
![2](https://user-images.githubusercontent.com/66586230/184272611-8f9763bb-f343-48ce-b696-b6eea630c999.png)
![3](https://user-images.githubusercontent.com/66586230/184272699-bc81aae4-32fa-435e-ab36-0eab8d339e03.png)

**Reply by wangwei2919** [reply]
I know, because I don't have the mxnet package... I thought pytorch was enough. Thank you very much for your guidance!
